File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Plants/Grass.java
package Organisms.Plants;

import Game.World;

import java.awt.*;
import java.io.Serializable;

public class Grass extends Plant implements Serializable {
    public Grass(World world,int x, int y)
    {
        super("G", "Grass", Color.green, 0, 0, 1, world,x,y);
    }
    @Override
    public Grass getChild(int x, int y)
    {
        return new Grass(world,x,y);
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Plants/grass.py
from Organisms.Plants.plant import Plant


class Grass(Plant):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Grass", 'G', 0, 0, world, 0, (0, 255, 0))

    def getchild(self, newposition):
        return Grass(newposition[0], newposition[1], self.world)
